src/types/commands.ts

```diff
@@ -673,6 +675,7 @@ export interface PasteFromOSClipboardCommand {
   type: "PASTE_FROM_OS_CLIPBOARD";
   target: Zone[];
   text: string;
+  textData: string;
```

Comment:
transform those two properties to only one `clipboardContent: ClipboardContent`
```suggestion
  clipboardContent: ClipboardContent
```